views/project/github.erb

```diff
@@ -32,7 +32,7 @@
               <th scope="col" class="py-3.5 pl-4 pr-3 text-left text-sm font-semibold text-gray-900 sm:pl-6">Runner</th>
               <th scope="col" class="px-3 py-3.5 text-left text-sm font-semibold text-gray-900">Repository</th>
               <th scope="col" class="px-3 py-3.5 text-left text-sm font-semibold text-gray-900">Label</th>
-              <th scope="col" class="px-3 py-3.5 text-left text-sm font-semibold text-gray-900">Vm</th>
+              <th scope="col" class="px-3 py-3.5 text-left text-sm font-semibold text-gray-900">State</th>
```

Comment:
is this an intentional renaming? It seems orthogonal from patch purpose ("don't crash")

Reply:
This is another outcome of moving it to model as `display_state`

From the commit message:
> With recent changes, this state isn't only the state of virtual machine. So I moved to the runner model as "display_state" similar to the other models